build-scripts/rcm-guest/publish-oc-binary.sh

```diff
@@ -12,13 +12,13 @@ extract() {
     mkdir macosx windows
     for arch in x86_64 ${ARCH}; do
         rpm=$(echo "$(rpm_name "${arch}")"*)
-        [ "${arch}" != x86_64 -a ! -e "${rpm}" ] && continue
+        if [[ "${arch}" != x86_64 ]] && [[ ! -e "${rpm}" ]]; then continue; fi
         mkdir "${arch}"
-        if [ "${arch}" != x86_64 ]; then
-            rpm2cpio ${rpm}* | cpio -idm --quiet ./usr/bin/oc
+        if [[ "${arch}" != x86_64 ]]; then
+            rpm2cpio "${rpm}" | cpio -idm --quiet ./usr/bin/oc
```

Comment:
@bbguimaraes not sure why the wildcard was removed here.

Reply:
It should have been moved to line 14, but was never removed from these two
lines.  This second wildcard is redundant, it can be seen in the logs that the
final file name is already contained in the variable.